ios/RNSentry.m

```diff
@@ -157,6 +157,16 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
             });
 }
 
+RCT_EXPORT_METHOD(fetchModules:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject)
+{
+    NSString *filePath = [[NSBundle mainBundle] pathForResource:@"modules" ofType:@"json"];
+    NSString* modulesString = [NSString stringWithContentsOfFile:filePath
+                                                  encoding:NSUTF8StringEncoding
+                                                     error:nil];
+    resolve(modulesString);
```

Comment:
Does this return `nil` if the file does not exist?

Reply:
Yes.